Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM ubuntu:trusty
+FROM ubuntu:bionic
 MAINTAINER Zach Halpern <zahalpern+github@gmail.com>
 
 RUN apt-get update && apt-get install -y software-properties-common
-RUN apt-add-repository ppa:ubuntu-sdk-team/ppa
-RUN add-apt-repository -y ppa:smspillaz/cmake-master
+#RUN add-apt-repository -y ppa:smspillaz/cmake-master
```

Comment:
I think we can remove that too as 18.04 LTS finally includs a more recent cmake (3.10), see https://packages.ubuntu.com/bionic/cmake

The old one only had 2.8 and we require [3.1](https://github.com/Cockatrice/Cockatrice/blob/master/CMakeLists.txt#L9)

Reply:
Removed that line.